deluge/tests/test_tracker_icons.py

```diff
@@ -24,6 +25,9 @@
 @pytest.mark.internet
 class TrackerIconsTestCase(BaseTestCase):
 
+    if windows_check():
+        skip = 'cannot use os.path.samefile to compair on windows(unix only)'
+
```

Comment:
Since this is in the tests, a more generic version would do: https://stackoverflow.com/a/8894488/175584